nodejs/week1/homework/warmup.js

```diff
@@ -0,0 +1,18 @@
+console.log("inside warmup file");
+
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+    getDiameter() { return this.radius * 2 }
+    getCircumference() { return this.getDiameter() * Math.PI }
+    getArea() { return Math.PI * this.radius * this.radius }
+}
+
+const circle1 = new Circle(10);
+const circle2 = new Circle(5);
+const circle3 = new Circle(12);
+
+console.log("Circle 1:", `diameter=${circle1.getDiameter()}, circumference=${circle1.getCircumference()}, area=${circle1.getArea()}`);
```

Comment:
You realized that `circumference and area` have many decimal after comma, there is a function that can help you to determine how many decimal you will have after comma. Try to put two decimal after comma.